app/main.py

```diff
@@ -2,0 +1,18 @@
+import os.path
+import shutil
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    src_file = parsed_command[1]
+    dst_file = parsed_command[2]
```

Comment:
Code Style issue: Here, you are assuming that the command will always have a source and destination file. It's better to add a check if the source and destination file are present in the command.